Source/Platform/DefaultOpenTK/Backend/Graphics/NativeWindow.cs

```diff
@@ -71,45 +72,44 @@ public NativeWindow(GraphicsMode mode, WindowOptions options)
 				if (DisplayDevice.Default != null)
 				{
 					options.Size = new Point2(
-						DisplayDevice.Default.Width, 
+						DisplayDevice.Default.Width,
 						DisplayDevice.Default.Height);
 				}
 			}
 
 			GameWindowFlags windowFlags = GameWindowFlags.Default;
 			if (options.ScreenMode == ScreenMode.FixedWindow)
+			{
 				windowFlags = GameWindowFlags.FixedWindow;
+			}
 			else if (options.ScreenMode == ScreenMode.Fullscreen)
+			{
 				windowFlags = GameWindowFlags.Fullscreen;
+			}
 
 			VSyncMode vsyncMode;
-			switch (options.RefreshMode)
-			{
-				default:
-				case RefreshMode.NoSync:
-				case RefreshMode.ManualSync:
-					vsyncMode = VSyncMode.Off;
-					break;
-				case RefreshMode.VSync:
-					vsyncMode = VSyncMode.On;
-					break;
-				case RefreshMode.AdaptiveVSync:
-					vsyncMode = VSyncMode.Adaptive;
-					break;
-			}
+			vsyncMode = GetVsyncModeFromOptions(options);
 
 			this.refreshMode = options.RefreshMode;
+
 			this.internalWindow = new InternalWindow(
 				this,
 				options.Size.X,
 				options.Size.Y,
 				mode,
 				options.Title,
-				windowFlags);
+				windowFlags
+			);
+
 			this.internalWindow.MakeCurrent();
+
 			this.internalWindow.CursorVisible = true;
+
```

Comment:
What's the purpose of the added newlines? Before the change, this block formed a semantic unit. Unless there is a specific reason to split this unit into multiple one-liners, it should remain as one.